CruCentralCoast/Events/EventsVC.swift

```diff
@@ -48,47 +57,102 @@ class EventsTVC: UITableViewController {
             }
             self.tableView.reloadData()
         }
-        
-        let longPressGestureRecognizer = UILongPressGestureRecognizer(target: self, action: #selector(handleLongPressGesture(gestureRecognizer:)))
-        longPressGestureRecognizer.minimumPressDuration = 0.3
+        createLongPressGesture()
+    }
+    
+    func createLongPressGesture() {
+        self.longPressGestureRecognizer = UILongPressGestureRecognizer(target: self, action: #selector(handleLongPressGesture(gestureRecognizer:)))
+        guard let longPressGestureRecognizer = self.longPressGestureRecognizer else {
+            return
+        }
+        longPressGestureRecognizer.minimumPressDuration = 0.05
+        longPressGestureRecognizer.allowableMovement = 0
         longPressGestureRecognizer.delegate = self
         self.tableView.addGestureRecognizer(longPressGestureRecognizer)
     }
     
     @objc internal func handleLongPressGesture(gestureRecognizer: UILongPressGestureRecognizer) {
         let touchPoint = gestureRecognizer.location(in: self.view)
         if gestureRecognizer.state == .began {
+            self.longPressCancelled = false
```

Comment:
we try to avoid adding state variables as much as we can like `longPressCancelled` and `longPressFailed` I'd recommend giving it your best shot to see if you can cover the gesture cancelled case without needing to track them like this.

Reply:
This is definitely something I want to fix in the future, but as for right now I am having trouble getting it to work. I completely agree, and this will be fixed when the gesture is finalized.